integration_test/itest/cluster.go

```diff
@@ -489,6 +537,20 @@ func (s *cluster) InstallTrafficManager(ctx context.Context, values map[string]s
 	return err
 }
 
+func (s *cluster) pullHelmChart(ctx context.Context, version string) (string, error) {
+	if err := Run(ctx, "helm", "repo", "add", "datawire", "https://app.getambassador.io"); err != nil {
+		return "", err
+	}
```

Comment:
if the repo already exists this call will fail; maybe check the error?

Reply:
I don't think that's true. You do get a message `datawire already exists with the same configuration, skipping`, but it doesn't fail.

Reply:
hmm okay, i've seen it fail in the makefile, but maybe that's something else